packages/babel-generator/test/fixtures/comments/async-class-method-computed-key-comment/output.js

```diff
@@ -1,3 +1,3 @@
 class C {
-  async [/* 2 */foo /* 3 */] /* 1 */ /* 4 */ /* 5 */() {}
+  async /* 1 */ /* 4 */ /* 5 */ [/* 2 */foo /* 3 */]() {}
```

Comment:
The original behavior is better for me, but I don't mind the new one.

Reply:
Yeah the problem is that there is no "correct" answer: due to how we mix inner comments, two of 1/4/5 will be in the wrong place.